app/models/ems_refresh/save_inventory.rb

```diff
@@ -1,12 +1,14 @@
 module EmsRefresh::SaveInventory
   def save_ems_inventory(ems, hashes, target = nil)
     case ems
-    when EmsCloud                                  then save_ems_cloud_inventory(ems, hashes, target)
-    when EmsInfra                                  then save_ems_infra_inventory(ems, hashes, target)
-    when ManageIQ::Providers::ConfigurationManager then save_configuration_manager_inventory(ems, hashes, target)
-    when ManageIQ::Providers::ContainerManager     then save_ems_container_inventory(ems, hashes, target)
-    when ManageIQ::Providers::NetworkManager       then save_ems_network_inventory(ems, hashes, target)
-    when ManageIQ::Providers::MiddlewareManager    then save_ems_middleware_inventory(ems, hashes, target)
+    when EmsCloud                                                  then save_ems_cloud_inventory(ems, hashes, target)
+    when EmsInfra                                                  then save_ems_infra_inventory(ems, hashes, target)
+    when ManageIQ::Providers::ConfigurationManager                 then save_configuration_manager_inventory(ems, hashes, target)
+    when ManageIQ::Providers::ContainerManager                     then save_ems_container_inventory(ems, hashes, target)
+    when ManageIQ::Providers::NetworkManager                       then save_ems_network_inventory(ems, hashes, target)
+    when ManageIQ::Providers::StorageManager::CinderStorageManager then save_ems_cloud_inventory(ems, hashes, target)
```

Comment:
Yes, it's better to have separate save_inventory for cinder. I'll add it accordingly. Thanks,